Website/htdocs/mpmanager/app/Http/Controllers/MainSettingsController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\ApiResource;
+use App\Models\MainSettings;
+
+class MainSettingsController extends Controller
+{
+    /**
+     *  @MainSettings
+     */
+
+    private $mainSettings;
+
+    public function __construct(MainSettings $mainSettings)
+    {
+        $this->mainSettings = $mainSettings;
+
+    }
+
+    public function index(): ApiResource
+    {
+        $mainSettings = MainSettings::all();
+        return new ApiResource($mainSettings);
+    }
+
+    public function update(MainSettings $mainSettings): ApiResource
+    {
+        $mainSettings = $this->mainSettings->find(request('id'));
```

Comment:
To update a model please use following schema; 

$model->update(['column' => $value]);

Reply:
get request parameter via request()->input('....')